lib/src/test/java/space/taran/arklib/lib/ResourceIdTest.kt

```diff
@@ -0,0 +1,26 @@
+package space.taran.arklib.lib
+
+import dev.arkbuilders.arklib.computeId
+import junit.framework.TestCase.assertEquals
+import org.junit.Before
+import org.junit.Test
+import kotlin.io.path.createTempFile
+import kotlin.io.path.fileSize
+import kotlin.io.path.writeText
+
+class ResourceIdTest {
+    @Before
+    fun before() {
+        System.loadLibrary("arklib")
+    }
+
+    @Test
+    fun resourceIdTest() {
+        val file = createTempFile()
```

Comment:
Although this file is temporary, it's a good practice to delete it at the end of the test.

Reply:
Sure